### PR TITLE
SEOImages: User Story 1463880, Part 18

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Files #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031700007/Report/

There are three image names identified by CATS as being too short. They should be handled at a later time as they are not part of the SEO image without hyphen work.

File: docs/framework/interop/default-marshaling-for-objects.md [hunk 1]
```diff
@@ -285,9 +285,9 @@ mo.SetVariant(new CurrencyWrapper(new Decimal(5.25)));
   
 <a name="cpcondefaultmarshalingforobjectsanchor6"></a>   
 ## Marshaling ByRef Variants  
- Although variants themselves can be passed by value or by reference, the **VT_BYREF** flag can also be used with any variant type to indicate that the contents of the variant are being passed by reference instead of by value. The difference between marshaling variants by reference and marshaling a variant with the **VT_BYREF** flag set can be confusing. The following illustration clarifies the differences.  
+ Although variants themselves can be passed by value or by reference, the **VT_BYREF** flag can also be used with any variant type to indicate that the contents of the variant are being passed by reference instead of by value. The difference between marshaling variants by reference and marshaling a variant with the **VT_BYREF** flag set can be confusing. The following illustration clarifies the differences:  
   
- ![Variant passed on the stack](./media/interopvariant.gif "interopvariant")  
+ ![Diagram that showsvVariant passed on the stack.](./media/default-marshaling-for-objects/interop-variant-passed-value-reference.gif)
```
Comment (mairaw) [inline docs/framework/interop/default-marshaling-for-objects.md]:
```suggestion
 ![Diagram that shows variant passed on the stack.](./media/default-marshaling-for-objects/interop-variant-passed-value-reference.gif)  
```

File: docs/framework/winforms/advanced/how-to-draw-a-line-with-line-caps.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ You can draw the start or end of a line in one of several shapes called line cap
   
  The following example draws a line with an arrowhead at one end and a round cap at the other end. The illustration shows the resulting line:  
   
- ![Pens](./media/pens4.gif "pens4")  
+ ![Illustration that shows a line with a round cap.](./media/how-to-draw-a-line-with-line-caps/line-cap-arrowhead-example.gif "pens4")
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-draw-a-line-with-line-caps.md]:
```suggestion
 ![Illustration that shows a line with a round cap.](./media/how-to-draw-a-line-with-line-caps/line-cap-arrowhead-example.gif)  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks good @damabe. A few comments to be addressed before we merge.

**Comment by damabe** [comment]
@mairaw Feedback incorporated. Thanks for the feedback. New CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031900024/Report/

There are three image names identified by CATS as being too short. They should be handled at a later time as they are not part of the SEO image without hyphen work.

**Review by mairaw — APPROVED** [review]
Thanks for applying my suggestions @damabe 👍

**Pull request merged**